plugins/plugin.proto

```diff
@@ -84,18 +96,29 @@ message PluginResponse {
   // status code.
   repeated string error = 1;
```

Comment:
Let's just make the compiler send a single model for now. That simplifies the plugin interface and covers plenty of use cases. I'll make the change to plugin.proto and propagate it through the samples.